manifests/bucketeer/charts/batch/values.yaml

```diff
@@ -220,3 +263,29 @@ processors:
   userEventPersister:
     flushSize: 200
     flushInterval: 5
+
+onDemandProcessors:
+  evaluationCountEventDWHPersister:
+    flushSize: 100
+    flushInterval: 10
+    flushTimeout: 10
+    project:
+    bigQueryDataSet:
+    bigQueryBatchSize:
+    timezone:
+  evaluationCountEventOPSPersister:
+    flushSize: 100
+    flushInterval: 10
+    flushTimeout: 10
+  goalCountEventDWHPersister:
+    flushSize: 100
+    flushInterval: 10
+    flushTimeout: 10
+    project:
+    bigQueryDataSet:
+    bigQueryBatchSize:
+    timezone:
+  goalCountEventOPSPersister:
+    flushSize: 100
```

Comment:
Please use the following default settings for all the new persisters.

```yaml
    flushSize: 20
    flushInterval: 5
    flushTimeout: 30
```

Reply:
Updated.